apps/root/src/pages/strategy-guardian-detail/strategy-management/components/fees/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { DisplayStrategy, FeeType } from 'common-types';
+import { FormattedMessage } from 'react-intl';
+import { ContainerBox, DividerBorder1, Skeleton, Typography } from 'ui-library';
+import useRawUsdPrice from '@hooks/useUsdRawPrice';
+import { isUndefined } from 'lodash';
+import { parseUsdPrice } from '@common/utils/currency';
+import { formatUnits, parseUnits } from 'viem';
+
+interface StrategyManagementFeesProps {
+  strategy?: DisplayStrategy;
+  feeType: FeeType;
+  assetAmount?: string;
+}
+
+const StrategyManagementFees = ({ strategy, feeType, assetAmount }: StrategyManagementFeesProps) => {
+  const [assetPrice, isLoadingPrice] = useRawUsdPrice(strategy?.asset);
+
+  const feeAmount = React.useMemo(() => {
+    const feePercentage = strategy?.guardian?.fees.find((fee) => fee.type === feeType)?.percentage;
+
+    if (!feePercentage || !assetAmount) return undefined;
+
+    const parsedAssetAmount = parseUnits(assetAmount, strategy.asset.decimals);
+    const feePercentageBigInt = BigInt(Math.round(feePercentage * 100));
+
+    return (parsedAssetAmount * feePercentageBigInt) / 100000n;
+  }, [strategy, feeType, assetAmount]);
+
```

Comment:
we could just return null if guardian does not exist or if feeamount is undefined